resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/circuitbreaker/monitoring/endpoint/CircuitBreakerChain.java

```diff
@@ -111,6 +112,26 @@ public void execute(Chain chain) throws Exception {
                             .event(c -> c.getEventType().name()).data(data));
                 ctx.render(events);
             });
+            chain1.get("hystrixStream/events", ctx -> {
```

Comment:
@RobWin do we want to name things hystrix - or name things turbine or a more generic name? The use case for all of this is that the streams are aggregated by turbine, which is polled by vizceral to provide real-time visualization of network traffic across all application instances.

Hystrix provided a streaming endpoint like this one - and Vijay is porting it forward for clients using resilience4j.

Reply:
@RobWin  Can you answer this? Otherwise I believe I will update with more documentation as pointed out by @drmaas  and this PR is good to merge.

Reply:
The streaming data format is turbine/hystrix specific, or?

Reply:
yes. They are intend for hystrix/turbine but can be used for any purpose. just enriching SSE with more data.. thats all